defaults/main.yml

```diff
@@ -13,6 +13,9 @@ nim_waku_node_tmp_path: '{{ nim_waku_cont_vol }}/data/tmp'
 #nim_waku_node_key: ~
 nim_waku_node_key_file_path: '{{ nim_waku_cont_vol }}/nodekey'
 
+# Waku Messages
+nim_waku_max_msg_size: 'size:150KiB'
```

Comment:
Is this param mandatory ? 
If not, should it be set at this value for all fleets ? 
If so then the `{% if nim_waku_max_msg_size is defined %}`  is not usefull
If not then it's comment this value,  so we know it exist and can add the value in role like `infra-nim-waku` or `infra-shards`